pkgs/by-name/ea/eapmd5pass/package.nix

```diff
@@ -0,0 +1,40 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  openssl_1_1,
```

Comment:
Actually, I think new packages depending on openssl 1.1 won't last long.

See #269713

Reply:
You are right. Maybe it makes more sense to close this then.